source/index.html

```diff
@@ -88,11 +90,11 @@ <h3 class="product-features__title">
           <li class="product-features__item">
             <h3 class="product-features__title">
               Заводит новых друзей
-            </h3>>
+            </h3>
             <p class="product-features__description">
               Ставьте лайки, комментируйте, делитесь фотографиями с друзьями и заводите новых
             </p>
-            <img class="product-features__img" src="https://via.placeholder.com/157x290" width="157" height="290" alt="Пример использования фильтра и комментариев в приложении Пинк.">
+            <img class="product-features__img" src="https://via.placeholder.com/157x287" width="157" height="280" alt="Пример использования фильтра и комментариев в приложении Пинк.">
```

Comment:
Так неправильно же )
Надо или так:
```suggestion
            <img class="product-features__img" src="https://via.placeholder.com/157x287" width="157" height="287" alt="Пример использования фильтра и комментариев в приложении Пинк.">
```

Ну или так:
```suggestion
            <img class="product-features__img" src="https://via.placeholder.com/157x280" width="157" height="280" alt="Пример использования фильтра и комментариев в приложении Пинк.">
```

Что в общем то не так и важно )